.github/workflows/cmake.yml

```diff
@@ -41,7 +44,16 @@ jobs:
     - name: Create Build Environment
       # Some projects don't allow in-source building, so create a separate build directory
       # We'll use this as our working directory for all subsequent commands
-      run: cmake -E make_directory ${{github.workspace}}/build
+      run: |
+          echo "tzdata tzdata/Areas select America" > ubuntu-preseed.txt
+          echo "tzdata tzdata/Zones/Etc select UTC" >> ubuntu-preseed.txt
+          echo "tzdata tzdata/Zones/America select New_York" >> ubuntu-preseed.txt
+          debconf-set-selections ubuntu-preseed.txt
```

Comment:
sounds like this also needs `sudo`?

Reply:
Removed, didn't work and should instead be fixed by removing the `setlocale` call in the unit test.